constants/taskRequests.ts

```diff
@@ -4,6 +4,11 @@ const TASK_REQUEST_STATUS = {
   PENDING: "PENDING",
   DENIED: "DENIED",
 };
+
+const TASK_REQUEST_UPDATE_ACTION = {
```

Comment:
+1

Reply:
These action is for updating the task request so keeping it the same.

Reply:
does task_request also have actions?

Reply:
- I think having a list of all actions related to task requests would make it easy for anyone new to the code to understand what various actions can we perform on a TR also managing it will be relatively easier as compared to actions that are spread out across the codebase.

Reply:
changed the name.